cmd/service_deploy.go

```diff
@@ -31,24 +36,41 @@ via the --image flag.
 The docker-compose.yml format is also supported using the --file flag.
 If -f is specified, the image and the environment variables in the
 docker-compose.yml file will be deployed.
+
+A task definition revision can be specified via the --revision flag.
+The revision number can either be absolute or a delta specified with a sign
+such as +5 or -2, where -2 is "2 configurations ago" from the current
+deployed revision.
 `,
 	Example: `
 fargate service deploy -i 123456789.dkr.ecr.us-east-1.amazonaws.com/my-service:1.0
 fargate service deploy -f docker-compose.yml
+fargate service deploy -r 37
 `,
 	Run: func(cmd *cobra.Command, args []string) {
 		operation := &ServiceDeployOperation{
 			ServiceName: getServiceName(),
+			Task:        getTaskName(),
```

Comment:
I don't think task should be required to deploy -i or -f (breaks backcompat).  If you don't specify it you get...
```shell
$ fargate service deploy -f docker-compose.yml
please specify task family using: fargate.yml, FARGATE_TASK envvar, or --task
```

Reply:
Good catch, fixed to grab the task family from the current task ARN